src/MessagingBubble/MessagingBubble.tsx

```diff
@@ -10,11 +10,11 @@ const cssClasses = {
 
 interface Props {
   className?: string;
-  content: React.ReactNode;
+  children: React.ReactNode;
```

Comment:
This is a small breaking change - I felt like it was more idiomatic to just use `children` for the content to display instead of another prop.